src/error.rs

```diff
@@ -24,27 +24,18 @@ pub enum Error {
 
 impl Error {
     pub fn description(&self) -> &str {
-        match *self {
-            #[cfg(feature = "std")]
-            Error::IO(_) => { "IO error" }
-            Error::Scroll(_) => { "Scroll error" }
-            Error::BadMagic(_) => { "Invalid magic number" }
-            Error::Malformed(_) => { "Entity is malformed in some way" }
-        }
+        "description() is deprecated; use to_string()"
```

Comment:
I don't know if any dependent crates using this method. But it might be deprecated in favor of `to_string`.

Reply:
That's not a good way to do deprecations. They need to be handled during compilation.

Reply:
Fixed in 2a0c80a.